Package.swift

```diff
@@ -62,6 +62,50 @@ if useSwiftCryptoV2 {
     swiftSettings.append(.define("CRYPTO_v2"))
 }
 
+let packageCollectionSigningTargets: [Target]
+// swift-crypto's Crypto module depends on CCryptoBoringSSL on these platforms only
+#if os(Linux) || os(Windows) || os(Android)
+packageCollectionSigningTargets = [
+    .target(
+        /** Package collections signing C lib */
+        name: "PackageCollectionsSigningLibc",
+        dependencies: [
+            .product(name: "Crypto", package: "swift-crypto"), // for CCryptoBoringSSL
+        ],
+        exclude: ["CMakeLists.txt"],
+        cSettings: [
+            .define("WIN32_LEAN_AND_MEAN"),
+        ]
+    ),
+    .target(
+         /** Package collections signing */
+         name: "PackageCollectionsSigning",
+         dependencies: [
+            "Basics",
+            .product(name: "Crypto", package: "swift-crypto"),
+            "PackageCollectionsModel",
+            "PackageCollectionsSigningLibc",
+         ],
+         exclude: ["CMakeLists.txt"],
+         swiftSettings: swiftSettings
+    ),
+]
+#else
+packageCollectionSigningTargets = [
+    .target(
+         /** Package collections signing */
+         name: "PackageCollectionsSigning",
+         dependencies: [
+            "Basics",
+            .product(name: "Crypto", package: "swift-crypto"),
+            "PackageCollectionsModel",
+         ],
```

Comment:
Can we always define the target but just add the dependency conditionally to avoid the duplication?